bin/ajax

```diff
@@ -769,6 +819,23 @@ def log_warning(message, priority='warning', run_id=None):
         log_coll.insert_one(warning_message)
     log.info(message)
 
+
+def set_state(state):
+    """Inform the bootstrax collection we're in a different state
+
+    if state is None, leave state unchanged, just update heartbeat time
+    """
+    ajax_state = dict(
+        host='ajax.' + hostname,
+        pid=os.getpid(),
+        time=now(),
+        state=state,
+        mode=f'clean {args.clean}',
+        production_mode=args.execute
+    )
+    bs_coll.insert_one(ajax_state)

```

Comment:
This would get updated frequently when cleaning. However, if ajax is sleeping, I don't think there is a lot of reason to update it right. 

The sleep time is set to 3600 s:
https://github.com/XENONnT/straxen/blob/0b7d7492e5f7aaf4e19f23a093f16aa07bf1012b/bin/ajax#L52